homeassistant/components/livisi/config_flow.py

```diff
@@ -74,6 +74,8 @@ async def create_entity(
         if (controller_data := controller_info.get("gateway")) is None:
             controller_data = controller_info
         controller_type = controller_data["controllerType"]
+        if controller_type == AVATAR:
+            controller_type = AVATAR_VERSION
```

Comment:
You need to have 100% coverage on config-flow.
Are the changes here required?
I think you can revert them and maybe implement in a dedicated PR.

Reply:
I reverted this change, I will do this in another pull request in the future. Thank you.